app/src/test/java/com/kickstarter/libs/utils/ConfigUtilsTest.kt

```diff
@@ -31,21 +31,15 @@ class ConfigUtilsTest : KSRobolectricTestCase() {
 
         assertEquals(JSONArray(), ConfigUtils.enabledFeatureFlags(ConfigFactory.configWithFeatureEnabled("ios_native_checkout")))
 
-        assertEquals(JSONArray().apply { put("android_go_rewardless_2") },
-                ConfigUtils.enabledFeatureFlags(ConfigFactory.configWithFeatureEnabled(FeatureKey.ANDROID_GO_REWARDLESS)))
+        assertEquals(JSONArray(),
+                ConfigUtils.enabledFeatureFlags(ConfigFactory.configWithFeaturesEnabled(mapOf(Pair(FeatureKey.ANDROID_NATIVE_CHECKOUT, false),
+                        Pair("ios_go_rewardless", true),
```

Comment:
Do we want to use the ios feature flags?

Reply:
This test is making sure we don't return any feature flags that start with `ios_`, only the ones that start with `android_`. I could rename it to `ios_feature_1`, I just used a name we were familiar with. What do you think?

Reply:
Ahh, thank you for clarifying! You can do something like `ios_feature_that_wishes_it_was_android` lol. You can keep it as is since it's just a test.